### DISCUSSION TITLE
How to set cropped YOUTUBEBB dataset downloaded from BaiduYun in crop511 dir?

### DISCUSSION THREAD

**Discussion by Holmes-GU** [STVIR/pysot#436] [General]
Since I directly download the cropped dataset YOUTUBEBB from BaiduYunn. So how should I do to set it to crop511 just like other three datasets do?

**Comment by wjh-scut** [comment]
Have you solved the problem? How can I use the cropped dataset YOUTUBEBB to generate train.json and val.json?
